tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -905,6 +905,10 @@ Status StorageManager::get_fragment_info(
   return array_close_for_reads(array_uri);
 }
 
+const std::map<std::string, std::string>& StorageManager::tags() const {
```

Comment:
since we don't need ordering (`map` is typically implemented with a red/black tree), `unordered_map` (implemented as a hash table) will perform significantly better. although the number of elements are small enough that this won't really matter.

Reply:
Fair point although the map will presumably always be very small. Made the change anyway.

Reply:
Thanks! Still LGTM.